src/jquery.multiselect.js

```diff
@@ -90,110 +140,130 @@
     * Performs initial widget creation
     * Widget API has already set this.element and this.options for us
     * All inserts into the DOM are performed at the end to limit performance impact
-    *   - Set the widget icons
-    *   - Assign header text values
+    *   - Build header links based on options and linkInfo object
     *   - Set UI effect speeds
     *   - Sets the multiselect ID using the global counter
     *   - Creates the button, header, and menu
     *   - Binds events for the widget
     *   - Calls refresh to populate the menu
     */
    _create: function() {
-
       var $element = this.element;
       var elSelect = $element[0];
       var options = this.options;
       var classes = options.classes;
-      var headerOn = options.header;
-      var checkAllText = options.maxSelected ? null : options.checkAllText;
-      // Do an extend here to address icons missing from options.iconSet--missing icons default to those in defaultIcons.
-      var iconSet = $.extend({}, defaultIcons, options.iconSet || {});
-      var uncheckAllText = options.uncheckAllText;
-      var flipAllText = options.flipAllText;
       var wrapText = options.wrapText || '';
 
+      // Do an extend here to address link info missing from options.linkInfo--missing info defaults to that in linkDefaults.
+      var linkInfo = ( this.linkInfo = $.extend(true, {}, linkDefaults, options.linkInfo || {}) );
+
+      // Helper function used below
+      function _linkHTML(linkTemplate, linkID) {
```

Comment:
It references linkInfo which is defined just above it.

Reply:
@mlh758 If you look closely, it uses the linkInfo variable just above it.  I originally had it outside of _create(), but I had to reference this.linkInfo.  Since _create() is the only thing using it, I just decided to pull it inside of _create().  

Would you prefer it be outside of _create()?  Keep in mind it will still need to use the linkInfo stuff somehow.

Reply:
_create is assigning it to this.linkInfo so the _linkHTML method would be able to access it that way. I would prefer to have it outside the _create method since it isn't a callback, and the data it's capturing in the closure is also accessible via `this`.

Reply:
Ok, I will move it back outside _create() then.